.circleci/config.yml

```diff
@@ -193,22 +191,61 @@ jobs:
       - store_test_results:
           path: /tmp/tests
 
+  test_package:
+    machine:
+      image: circleci/classic:201711-01
+    working_directory: /tmp/src/niworkflows
+    steps:
+      - checkout
+      - run: pyenv local 3.5.2
+      - run:
+          name: Install build depends
+          command: python3 -m pip install "setuptools>=30.3.0" "pip>=10.0.1" twine docutils
+      - run:
+          name: Build and check
+          command: |
+            python3 setup.py check -r -s
+            python3 setup.py sdist
+            python3 -m twine check dist/*
+      - run:
+          name: Validate version
+          command: |
+            THISVERSION=$( python3 get_version.py )
+            python3 -m pip install dist/*.tar.gz
+            mkdir empty
+            cd empty
+            INSTALLED=$( python3 -c 'import niworkflows; print(niworkflows.__version__)' )
```

Comment:
This fails if we build and install from a wheel... I'm not sure why. Perhaps we need to upgrade wheel?

Reply:
I honestly have no clue